email_campaigns/email_to_attendee.txt

```diff
@@ -0,0 +1,77 @@
+Subject: You’re attending the mentored sprint for diverse beginners at PyCon US!
+
+{Full name},
+
+Thank you for signing up to the mentored sprint for diverse beginners at PyCon US.
+We’re pleased to welcome you and help you contribute to open source.
+
+The mentored sprint will take place on **Saturday, May 4th 2:30 PM - 6:30 PM. Room 25C**
+
+Some useful information:
+
+- The mentored sprint for diverse beginners is organized as part of PyCon US. You will need
+  to be registered for PyCon in order to attend the mentored sprint. If you have not registered,
+  please do so as soon as possible, because PyCon is a sell out year after year.
+  
+- We’re excited to share that 11 open source projects have signed up to mentor! They have
+  prepared a list of beginner friendly issues and contributing guidelines. Please check out the
+  list of projects here: https://us.pycon.org/2019/hatchery/mentoredsprints/ 
```

Comment:
A total nit, but is it possible to deep link to the relevant paragraph? Otherwise there's a lot of scrolling involved to find the right information.

I'm not sure if the PyCon blog format supports this.

Reply:
Changed to https://mentored-sprint-for-diverse-beginners.readthedocs.io/projects_list.html